seedemu/core/InternetExchange.py

```diff
@@ -19,7 +19,7 @@ class InternetExchange(Printable, Configurable):
     __rs: Node
     __name: str
 
-    def __init__(self, id: int, prefix: str = "auto", aac: AddressAssignmentConstraint = None):
+    def __init__(self, id: int, prefix: str = "auto", aac: AddressAssignmentConstraint = None,create_rs = True):
```

Comment:
Would you mind adding a description of the `create_rs` parameter to the docstring?